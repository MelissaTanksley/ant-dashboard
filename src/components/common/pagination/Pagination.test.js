import React from 'react';
import { shallow } from 'enzyme';
import PaginationComponent from './pagination';

describe('<PaginationComponent />', () => {
    const wrapper = shallow(<PaginationComponent />);
    it('PaginationComponent matches snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    })
})