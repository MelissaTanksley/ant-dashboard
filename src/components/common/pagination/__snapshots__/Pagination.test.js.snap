// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PaginationComponent /> PaginationComponent matches snapshot 1`] = `
<div>
  <Pagination
    defaultCurrent={1}
    prefixCls="ant-pagination"
    selectPrefixCls="ant-select"
    total={50}
  />
</div>
`;
